app/templates/macros/toolkit_forms.html

```diff
@@ -129,7 +134,8 @@
     hint=(question_content.hint or '')|markdown,
     name=question_content.id,
     value=service_data[question_content.id],
-    error=errors.get(question_content.id)['message']
+    error=errors.get(question_content.id)['message'],
+    question_number=question_number
```

Comment:
Small tip for the future: macro arguments exist in the same context as other variables, so if the variable name and value that included template expects match the macro argument, there's no need to pass it in `with`, since it's available anyway.